integration_tests/test_str_attributes.py

```diff
@@ -366,7 +366,69 @@ def is_space():
     s = ""
     assert s.isspace() == False
 
+def is_alnum():
+    a: str = "helloworld"
+    b: str = "hj kl"
+    c: str = "a12(){}A"
+    d: str = " "
+    e: str = ""
+    f: str = "ab23"
+    g: str = "ab2%3"
+    res: bool = a.isalnum()
+    res2: bool = b.isalnum()
+    res3: bool = c.isalnum()
+    res4: bool = d.isalnum()
+    res5: bool = e.isalnum()
+    res6: bool = f.isalnum()
+    res7: bool = g.isalnum()
+
+    assert res == True
```

Comment:
You can also print the obtained result values before the assert. That helps in debugging if there is any failure later.

Reply:
Ok, I'll add print statements. Please let me know why a particular CI check fails. It's related to PIP installable packages.

Reply:
>Please let me know why a particular CI check fails. It's related to PIP installable packages.

It is probably a fluke.